frontend/src/queries/example.ts

```diff
@@ -0,0 +1,62 @@
+import { query2dcids, Query, CategoryType } from "./query"
+import { Dcid } from "./dcids"
+
+import { DCIDS } from "./example_dcids"
+
+
+const Categories = {
+    'gender': new Set<string>([
+        'Male',
+        'Female',
+    ]),
+    'ethnicity': new Set<string>([
+        'HispanicOrLatino',
+        'OtherPacificIslander',
+        'AmericanIndianOrAlaskaNative',
+        'Asian',
+        'Black',
+        'TwoOrMoreRaces',
+        'WhiteAlone',
+    ]),
+    'occupation': new Set<string>([
+        'SOCComputerMathematicalOccupation',
+        'SOCEngineersOccupation',
+        'SOCLifePhysicalSocialScienceOccupation',
+        'SOCLifeScientistsOccupation',
+        'SOCMathematicalScienceOccupation',
+        'SOCPhysicalScientistsOccupation',
+        'SOCPsychologistsOccupation',
+        'SOCSocialScientistsRelatedWorkersOccupation',
+        'ScienceAndEngineeringOccupation',
+        'ScienceAndEngineeringRelatedOccupation',
+        'NonScienceAndEngineeringOccupation',
+    ]),
+}
+
+// Dimensions to Category mapping.
+const Dim2Cat = {}
+// console.log(Categories)
+
+Object.keys(Categories).forEach((key) => {
+    console.log(key)
+    const values = Categories[key]
+    values.forEach((value) => {
+        Dim2Cat[value] = key
+    })
+})
+// console.log(Dim2Cat)
+
+const Prefix = 'Count_Person_ScienceAndEngineeringRelatedMajor_EducationalAttainmentDoctorateDegree'
```

Comment:
I believe your dcid slicing logic will be able to handle this, but want to confirm we're on the same page that
the prefix won't be static & that the whole dcid will need to be generated in alphabetical order of its properties (not the values of the properties).

Reply:
Are these not the DCIDs we're going to use for querying? If they are, they're not in alphabetical order.

Reply:
Aligned offline, these are the real DCIDs and they're sorted based on the convention at https://docs.google.com/document/d/1YtmQu_gOXENt4W2lmlK-8Mtnw_XB3kZlLBjSN-gT0oE/preview#bookmark=kix.bdksa8hbv3si.